doc/query.md

```diff
@@ -39,25 +39,30 @@ Secondly, these documents are in the datastore:
 ```java
 { "name": "mike", 
   "age": 12, 
-  "pet": {"species": "cat"} };
+  "pet": {"species": "cat"},
+  "comment": "Mike goes to middle school and likes reading books." };
 
 { "name": "mike", 
   "age": 34, 
-  "pet": {"species": "dog"} };
+  "pet": {"species": "dog"},
+  "comment": "Mike is a doctor and likes reading books." };
 
 { "name": "fred", 
   "age": 23, 
-  "pet": {"species": "cat"} };
+  "pet": {"species": "cat"},
+  "comment": "Fred works for a startup out of his home office." };
 ```
 
-### Creating indexes
+### Indexes
```

Comment:
I think a slight reorganization of the beginning of this "Usage" section would make things more logical (and perhaps also some work on the first five sentences of "Usage" -- different PR?) This "Indexes" heading seems out of place here. Perhaps it should go above the line "Firstly, we set up an IndexManager object, im, as follows." and change it from "Indexes" to "Index Management".